lib/onebox/engine/allowlisted_generic_onebox.rb

```diff
@@ -256,6 +256,15 @@ def data
             d[:data_1] = Onebox::Helpers.truncate("#{d[:price_currency].strip} #{d[:price_amount].strip}")
           end
 
+          skip_missing_tags = [:video]
+          d.each do |k, v|
+            next if skip_missing_tags.include?(k)
+            if v == nil || v == ''
+              errors[k] ||= []
+              errors[k] = 'is blank'
```

Comment:
I think it was supposed to be:

```suggestion
              errors[k] ||= []
              errors[k] << 'is blank'
```

😉

Reply:
Good catch!